runtime/common/src/main/java/edu/snu/onyx/runtime/common/message/ncs/NcsMessageSender.java

```diff
@@ -27,13 +27,13 @@
 
   @Override
   public void send(final ControlMessage.Message message) {
-    LOG.debug("send: {}", message);
+    LOG.trace("send: {}", message);
```

Comment:
If you'd like to change this, please change the logging way of `NcsMessageEnvironment`, `GrpcMessageClient`, `GrpcMessageContext`,... and so on also.

Reply:
Sure.